### ISSUE TITLE
sanding success

### ISSUE THREAD

**Issue by lizzieinvancouver** [temporalecologylab/TreeRings#16]
Neil Pederson waxed on about sanding to me when I met him. What's our workflow? He recommended:

Start at and advance though these grits: 
- 120 (or 180)
- 220 (0r 240)
- 320 (some belts are 400, that's probably okay)
- Then go to one of the below and do it by hand. 

Good brands for small/fine sand paper

- 600 Hermes
- 3M micron (1200 grit should be enough) ... Look up 3M polishing paper and try those.

**Comment by DeirdreLoughnan** [comment]
@lizzieinvancouver @wangxm-forest  @afong3 I completed the first stage of the cookie sanding. 

1. I had and unforeseen issue with the 3D printed tray for the cores. The standard method is to use regular Elmer's water soluble glue (so you can fix the core positioning), but when I tried sanding it the cores came unstuck and were flung all over. I am not sure if the glue just sticks better to the wooden trays or if the trays just need to be deeper so more of the core is embedded. We could try other glues, but would have less room to correct any mistakes.

2. The sanding for our 93 cookies for the 120 grit took me about 4h from setup to cleanup. I think this will be one of the longest stages as you are sanding down the grooves left by the saws. But it is not a fast task.

3. I think in future having thicker cuttings would be better, some of the cookies are pretty small are are hard to hold.

**Comment by lizzieinvancouver** [comment]
@DeirdreLoughnan Amazing! And omigod -- I am so sorry to hear about (1). We definitely need some other glue and test cores, before we try the 3D trays again. Can we order wooden mounts anywhere (and/or put in an order now with woodshop)? I think Ailene has some she will give us that I can pick up in June ... 

Good to know on (3). 

Thank you for all your work on this!

**Comment by DeirdreLoughnan** [comment]
@lizzieinvancouver @wangxm-forest 

I can look into purchasing some from the woodshop. If Ailene has some that might be fastest and cheapest, but it would be good to ask if she knows the size of corer she used. I was given a free board from one of Lori's students, but our cores are too small. 

@wangxm-forest could you email Ailene and ask about her trays?

**Comment by afong3** [comment]
@DeirdreLoughnan 

Sorry the 3D trays didn’t work out as planned! I can make a couple of trays with varied depths / materials to test adhesion. Would debarked twigs work as a decent enough proxy to a core? I can get a couple different adhesives to test as well.

**Comment by DeirdreLoughnan** [comment]
Thanks @afong3! I imagine twigs could work, but we could also try to get some usable samples for the image stitching. I was going to chat with @wangxm-forest about taking some cores. It could be a good opportunity for people on her field team to get experience hand coring trees.

**Comment by lizzieinvancouver** [comment]
> Would debarked twigs work as a decent enough proxy to a core? I can get a couple different adhesives to test as well.

@DeirdreLoughnan @afong3 It occurred to me that the dendro people may use Elmer's because it is water soluable even after drying.... I wonder if they want to make sure they can take cores off later. We may not care to do that, but we should find out if they do that and why so we make sure it does not apply to us.

**Comment by DeirdreLoughnan** [comment]
@lizzieinvancouver yes, that is correct. Based on my readings it is to fix your mistakes if the cores are not mounted in the correct orientation. This might be a concern at the start when we are learning, but there are features to look for to make sure you are mounting cores properly the first time.

**Comment by DeirdreLoughnan** [comment]
@lizzieinvancouver a quick update, I finished the two additional grades of sanding. It was a little bit faster, but it still took 6 hours in total. 

Today I met with Ken and Britany to try our hand at the final two grades of hand sanding. In discussing the sample prep with @afong3 (see issue #3), it seems important that samples be flat. Many of our samples were cut at an angle and some have some slight lips at the edges. Some of our common garden samples will likely need to be sanded again to flatten them more.

**Comment by lizzieinvancouver** [comment]
@DeirdreLoughnan Thanks for working on this! I wonder if any sort of planing device would help? I know one lab mentioned a Japanese planer as being good (I assume a hand planer) but I do not recall who. Everyone at WSL uses a microtome, but I assume it is some giant microtome that no one else on earth can afford (but I could be wrong).

**Comment by afong3** [comment]
@DeirdreLoughnan @lizzieinvancouver I ordered a tabletop belt sander and it's now in the lab. Is there some sort of training I need to do to access a room that is capable of handling the volume of saw dust that will be created?

**Comment by lizzieinvancouver** [comment]
> Is there some sort of training I need to do to access a room that is capable of handling the volume of saw dust that will be created?

Hmm, there used to be a room for potting plants ... and there is wood products space, but we're probably not allowed to just rock up and use our own equipment there. I will try to ask around and maybe @DeirdreLoughnan has ideas.

**Comment by lizzieinvancouver** [comment]
@DeirdreLoughnan offered (abridged to remove email info):

> In answer to your git issue about setting up the belt sander, the person to speak to is [LE]. He is the Technical Operations Coordinator for CAWP.
> 
> They have a room where the [other labs keep who work on tree rings] their equipment that [LE] does the initial training for. After that you have to fill out the CAWP safety documents and send them to [IF] to get room tap access.
